.github/workflows/ci.yml

```diff
@@ -38,16 +39,19 @@ jobs:
       fail-fast: false
 
     steps:
-      - uses: actions/checkout@v2
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1.4.4
+      - uses: actions/checkout@v3
+      - name: Use Node.js 14
+        uses: actions/setup-node@v3
         with:
-          node-version: 14.x
+          node-version: 14
+          check-latest: true
       - name: Setup Java JDK
-        uses: actions/setup-java@v1.4.3
+        uses: actions/setup-java@v3
```

Comment:
This change is likely breaking CI

Reply:
In what way? Due to the already specified Java version not being available or?

Reply:
The v3 implies a breaking change. Reverting this change may make the CI pass if the breakage here[1] is due to the change in API upstream. 

[1] https://github.com/PrismarineJS/mineflayer/actions/runs/4051469496/jobs/6969827556#step:4:1

Reply:
The latest versions of the `actions/setup-java` action rely on different distributions mentioned [here](https://github.com/actions/setup-java#supported-distributions).

And I already mentioned that the already specified Java version isn't available in this comment: https://github.com/PrismarineJS/mineflayer/pull/2922#issuecomment-1409849423

So which Java version do you think is appropriate here? You can also check all the other distributions if the one needed isn't available.

Reply:
What I meant in my comment above was to revert this change. The CI on master is currently passing. It seems likely this change is breaking  CI on this PR.

Reply:
I get that, I mean like reverting that change isn't ideal because that the older version of `actions/setup-java` is also subject to this point I made earlier:

> The older versions use outdated commands and requests that GitHub has deprecated (such as the set-output command being used by the actions/* actions) and will soon be disabled, rendering the actions unusable.

That's why I want to update it 😅.

Reply:
Java 8 should be synonymous with 1.8, you can try replacing 1.8 here with 8

Reply:
Got it, though some of the Minecraft server tests are using Java v16 which is generally unavailable since it's not LTS.

Is there a reason Java v17 isn't being used for all of them? I'm pretty sure those Minecraft versions work fine with Java v17.

Reply:
The CI passed with Java v17 for that Minecraft version.